### PR TITLE
fix: Use new version of the connector.

### PR DESCRIPTION
**Pull request by mfornet**
This PR was created to trigger CI (and integration tests) with the new version of the connector

TODO: Don't use current pinned commit from token-connector, instead wait until it is merged into master

### CODE REVIEW

**Comment by djsatok** [comment]
Closes #519

**Review by djsatok — APPROVED** [review]
Looks good, please check the commits

File: contracts/near/admin-controlled/Cargo.toml [hunk 1]
```diff
@@ -8,4 +8,4 @@ edition = "2018"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-near-sdk = "0.11.0"
+near-sdk = { git = "https://github.com/near/near-sdk-rs", rev = "613adcab6322e4f830efb4d5bde2a810ae642158" }
```
[the remark this answers is not in the document]
Comment (mfornet) [reply contracts/near/admin-controlled/Cargo.toml:11]: It is the right commit (the first in this PR: https://github.com/near/near-sdk-rs/pull/325) It is already merged, after release we will be able to update.

File: cli/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "rainbow-bridge-near2eth-block-relay": "1.0.0",
     "rainbow-bridge-watchdog": "1.0.0",
     "rainbow-bridge-testing": "1.0.0",
-    "rainbow-token-connector": "near/rainbow-token-connector#0646850267c42057e92f2d670eebe79c9c9211f9",
+    "rainbow-token-connector": "near/rainbow-token-connector#6855221404f81f987fcf1db9a860b2345149531a",
```
[the remark this answers is not in the document]
Comment (mfornet) [reply cli/package.json:37]: atm of writing it, that PR haven't land, let's update.

Comment (mfornet) [inline cli/package.json:37]:
```suggestion
    "rainbow-token-connector": "near/rainbow-token-connector#d6022357e8e59ea790ff52af80df40f6036d1ec9",
```

**Comment by mfornet** [comment]
This PR already contains #525 and #524

**Pull request merged**